package.json

```diff
@@ -51,7 +51,7 @@
     "jest-mock-extended": "^2.0.4",
     "node-mocks-http": "^1.11.0",
     "prettier": "2.5.1",
-    "prisma": "^3.7.0",
+    "prisma": "^3.15.2",
```

Comment:
Why bump?

Reply:
prisma client is at 3.15 so we wanted to update